ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -850,6 +850,9 @@ objects:
           mkdir -p /tmp/artifacts/must-gather
           queue /tmp/artifacts/must-gather/must-gather.log oc --insecure-skip-tls-verify adm must-gather --dest-dir /tmp/artifacts/must-gather
 
+          echo "Removing ssh-bastion ..."
+          queue /dev/null oc --insecure-skip-tls-verify --request-timeout=5s delete project testing-ssh-bastion
```

Comment:
Why are you doing this?  Why isn't this being torn down by the cluster tear down?

Reply:
It was requested in https://github.com/openshift/release/pull/4161#pullrequestreview-254023886

Reply:
Service load balancers aren't leaked.